app/services/travis_script.rb

```diff
@@ -0,0 +1,14 @@
+require 'uri'
+require 'net/http'
+require 'json'
+
+uri = URI.parse("https://api.travis-ci.org/repo/ruby-bench%2Fruby-bench-docker/requests")
+https = Net::HTTP.new(uri.host,uri.port)
+https.use_ssl = true
+request = Net::HTTP::Post.new(uri.path)
+request["Content-Type"] = 'application/json'
+request["Accept"] = 'application/json'
+request["Travis-API-Version"] = '3'
+request["Authorization"] = 'token smEOuqMAAUcwLHBnFjnJkA'
+request.body = {'request' => {'branch' => 'master'}}.to_json
+response = https.request(request)
```

Comment:
I did not get this. Currently we get a success. Do I need to check for something?